src/expandableCalendar/index.tsx

```diff
@@ -98,7 +98,7 @@ const headerStyleOverride = {
  */
 
 const ExpandableCalendar = (props: ExpandableCalendarProps) => {
-  const {date, setDate, numberOfDays = 1} = useContext(Context);  
+  const {date, setDate, numberOfDays = 1, leftInset = 72} = useContext(Context);  
```

Comment:
We're setting the `72` default values in several places. 
if we'll need to change it in the future we might miss one instance and have a bug. 
Can we maybe set its default once in the context value?

Reply:
Done :)